src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,45 @@
 import React from 'react';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+
+import { FullPost } from '../../types/FullPosts';
+
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  post: FullPost
+}
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    comments,
+    user,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        <p>
+          {' Posted by  '}
```

Comment:
Now you will render `Posted by` even in case you don't have a user, let us fix it